hexrd/ui/hexrd_config.py

```diff
@@ -1346,7 +1344,7 @@ def validate_overlays(self):
 
     @property
     def visible_polar_masks(self):
```

Comment:
Maybe we should change this name to `visible_masks`? But that can be done another time...

Reply:
It was a quick and easy fix so I updated it